lib/dealership.rb

```diff
@@ -0,0 +1,20 @@
+class Dealership
+  attr_reader :name, :address, :inventory
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def inventory_count
+    inventory = 0
+  end
+
+
+  def add_car(car)
+    @inventory << car
+    inventory.count
+  end
```

Comment:
Adding the car is right! Since we derive the inventory_count from the @inventory we only need the first line

```suggestion
  def add_car(car)
    @inventory << car
  end
```